src/com/model/BanknotesHistory.java

```diff
@@ -0,0 +1,49 @@
+package com.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BanknotesHistory {
+    public Map<Integer, BanknoteHistoryData> banknotesHistoryMap = new HashMap<>();
+
+    // Init historyMap
```

Comment:
Well, this is a good attempt to create a Javadoc ;) but an actual javadoc must follow the format:
```
/**
 * Description what a method does
 * @param key What the argument means
 * @param value What the argument means
 * @return What is being returned (unless the method is void
 */
```